help/destinations/catalog/advertising/tradedesk-emails-update

```diff
@@ -0,0 +1,138 @@
+---
```

Comment:
Comment about the name of the file itself (tradedesk-emails-update). Considering that we are updating an existing file and not creating a new documentation page, we should leave the name of the file as the old `tradedesk-emails.md`